### DISCUSSION TITLE
keine Funktion der Dropdown Senderliste im Webinterface

### DISCUSSION THREAD

**Discussion by mcbo007** [beabel/radiowecker#56] [General]
Hi,

ich sehe gerade im Webinterface (sehr gut gelungen),
das die Dropdown Senderliste im ersten Tab keine Funktion hat.
Das Feld ist grau und kann nicht ausgewählt werden.
Hier soll man doch sicher den aktuellen Sender umstellen können?

Liegt das an meinem Browser oder ist das ein Bug?

**Comment by beabel** [comment]
Nein dort wird nur der aktuelle angezeigt.
Ich habe dafür jedoch das bereits vorhandene drop-down genutzt.
Weiterschalten kannst du nur mit <>

Vielleicht kommt in späteren Versionen noch eine bessere Umschaltung, Mal sehen.
Zuerst möchte ich jedoch den Code im TFT Display etwas besser organisieren.
Bringt zwar keine Benutzer Vereinfachung, jedoch am Ende besser wartbaren und erweiterbare code.
